web-registry/src/lib/db/api/metadata-graph.ts

```diff
@@ -1,19 +1,52 @@
+import { QueryClientImpl } from '@regen-network/api/lib/generated/regen/data/v1/query';
```

Comment:
the most important changes to review are in this file
the rest of the changes are mostly about wiring up all getMetadata calls with the data module query client